armory/data/url_checksums/carla_over_obj_det_dev.txt

```diff
@@ -0,0 +1 @@
+armory-public-data carla/carla_over_od_dev_1.0.0.tar.gz 61751299 ee6fbd0f00cf88aaef3ae097e9d661d52b0d97e5688098d342dad4780f931c8a
```

Comment:
Should this be changed to the following? I was unsure why the old version did not have the full url and why the cached versions never do either.
```suggestion
https://armory-public-data.s3.us-east-2.amazonaws.com/carla/carla_over_od_dev_1.0.0.tar.gz 61751299 ee6fbd0f00cf88aaef3ae097e9d661d52b0d97e5688098d342dad4780f931c8a
```

Reply:
I don't have the answer to this question because it's always done by someone at TwoSix.

Reply:
Should have put this tag here before, @davidslater thoughts?

Reply:
There are two types of checksums:
`url_checksums` are meant for the original raw data, and have the full `https://...` URL. These are what TFDS uses to build the datasets.
`cached_s3_checksums` have the fully processed TFDS data, and have the `armory-public-data...` format.

Since this one references the raw dataset, it should have the `https://...` format.